packages/typespec-ts/test/commands/cadl-ranch-list.js

```diff
@@ -880,6 +880,10 @@ export const modularTsps = [
     outputPath: "payload/media-type",
     inputPath: "payload/media-type"
   },
+  {
+    outputPath: "payload/multipart",
+    inputPath: "payload/multipart"
+  },
```

Comment:
maybe also add this in azureModularTsp? I would assume there's no difference between azure and non-azure regarding mfd?

Reply:
Yes, should be no difference. Added to the Azure list as well.